hydrus/data/crud.py

```diff
@@ -186,7 +189,39 @@ def insert(object_: Dict[str, Any], session: scoped_session,
                 # Adds new Property
                 session.close()
                 raise PropertyNotFound(type_=prop_name)
-
+            # For insertion in III through link
+            if prop_name in link_props:
+                regex = r'[a-z0-9]{8}-([a-z0-9]{4}-){3}[a-z0-9]{12}'

```

Comment:
Add proper comments what are you trying to do with regex and why

Reply:
Preprocessing of link properties change property value to instance_id in case of link to collection class and class_type in case of link to non-collection class, I am adding comments.